tests/test_order_SolidLiquid.py

```diff
@@ -51,6 +52,36 @@ def test_identical_environments(self):
                 assert comp.cluster_sizes[0] == len(positions)
                 npt.assert_array_equal(comp.num_connections, 12)
 
+    def test_l_zero(self):
+        from scipy.special import sph_harm
```

Comment:
In another test function (for Steinhardt) we use `pytest.importorskip` but here we just import. Is there a reason we switch between the two.

Reply:
No reason, this was older and is now removed.